rocAL/rocAL/include/augmentations/geometry_augmentations/node_crop.h

```diff
@@ -26,24 +26,22 @@ THE SOFTWARE.
 #include "parameter_crop_factory.h"
 #include "parameter_rocal_crop.h"
 
-class CropNode : public Node
-{
+class CropNode : public Node {
 public:
     CropNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs);
     CropNode() = delete;
+    ~CropNode();
```

Comment:
Why this has been added?

Reply:
Introduced to release crop tensor and ptr